### ISSUE TITLE
import_pbp_data not working

### ISSUE THREAD

**Issue by rjthompson2** [cooperdff/nfl_data_py#33]
When I run:
nfl.import_pbp_data([2021])
I get the following error:
Data not available for 2021

**Comment by rjthompson2** [comment]
No, I thought the columns were optional.

Also I'm not sure but I think the underlying issue might be:
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)>

**Comment by cooperdff** [comment]
Yes they are optional, specifying them is causing a problems right now so wanted to confirm that wasn't the issue.

Can you tell me what version of python and nfl_data_py you're running?

**Comment by rjthompson2** [comment]
I'm on python 3.10.6 and nfl-data-py 0.3.0

**Comment by cooperdff** [comment]
Okay I just set up a test env with the same python and nfl_data_py versions and I'm not getting any errors. Could you run conda list in your environment and share your full package list with me?

**Comment by rjthompson2** [comment]
Sure! Here it is:
# Name                    Version                   Build  Channel
_ipyw_jlab_nb_ext_conf    0.1.0                    py38_0  
anaconda                  2021.05                  py38_0  
backend                   0.1.0                    pypi_0    pypi
backports                 1.0                pyhd3eb1b0_2  
black                     22.6.0                   pypi_0    pypi
blas                      1.0                         mkl  
blosc                     1.21.0               h2842e9f_0  
boto                      2.49.0                   py38_0  
boto3                     1.20.42                  pypi_0    pypi
botocore                  1.23.42                  pypi_0    pypi
bottleneck                1.3.2            py38hf1fa96c_1  
brotlipy                  0.7.0           py38h9ed2024_1003  
builder                   0.1.0                    pypi_0    pypi
bzip2                     1.0.8                h1de35cc_0  
c-ares                    1.17.1               h9ed2024_0  
ca-certificates           2021.4.13            hecd8cb5_1  
cachetools                5.0.0                    pypi_0    pypi
carta-client              1.0.0                    pypi_0    pypi
certifi                   2020.12.5        py38hecd8cb5_0  
cffi                      1.14.5           py38h2125817_0  
chardet                   3.0.4                    pypi_0    pypi
click                     8.1.3                    pypi_0    pypi
cloudpickle               1.6.0                      py_0  
clyent                    1.2.2                    py38_1  
colorama                  0.4.4              pyhd3eb1b0_0  
commonmark                0.9.1                    pypi_0    pypi
conda                     4.13.0           py38hecd8cb5_0  
conda-build               3.21.4           py38hecd8cb5_0  
conda-content-trust       0.1.1              pyhd3eb1b0_0  
conda-env                 2.6.0                         1  
conda-package-handling    1.7.3            py38h9ed2024_1  
conda-repo-cli            1.0.4              pyhd3eb1b0_0  
conda-token               0.3.0              pyhd3eb1b0_0  
conda-verify              3.4.2                      py_1  
contextlib2               0.6.0.post1                py_0  
cramjam                   2.5.0                    pypi_0    pypi
cryptography              3.4.7            py38h2fd3fbb_0  
curl                      7.71.1               hb0a8c7a_1  
cycler                    0.10.0                   py38_0  
cython                    0.29.23          py38h23ab428_0  
cytoolz                   0.11.0           py38haf1e3a3_0  
dask                      2021.4.0           pyhd3eb1b0_0  
dask-core                 2021.4.0           pyhd3eb1b0_0  
dbus                      1.13.18              h18a8e69_0  
decorator                 5.0.6              pyhd3eb1b0_0  
defusedxml                0.7.1              pyhd3eb1b0_0  
deprecated                1.2.13                   pypi_0    pypi
diff-match-patch          20200713                   py_0  
distlib                   0.3.4                    pypi_0    pypi
distro                    1.6.0                    pypi_0    pypi
django                    4.0.6                    pypi_0    pypi
django-admin              2.0.1                    pypi_0    pypi
django-excel-base         1.0.4                    pypi_0    pypi
django-excel-response2    3.0.2                    pypi_0    pypi
django-six                1.0.4                    pypi_0    pypi
docopt                    0.6.2                    pypi_0    pypi
docutils                  0.17             py38hecd8cb5_1  
ecapture                  2.0.2                    pypi_0    pypi
entrypoints               0.3                      py38_0  
et_xmlfile                1.0.1                   py_1001  
expat                     2.3.0                h23ab428_2  
fabric                    2.6.0                    pypi_0    pypi
fantasyfootball           0.1.0                    pypi_0    pypi
fastcache                 1.1.0            py38h1de35cc_0  
fastparquet               0.8.2                    pypi_0    pypi
filelock                  3.4.2                    pypi_0    pypi
flake8                    3.9.0              pyhd3eb1b0_0  
flask                     1.1.2              pyhd3eb1b0_0  
flask-sqlalchemy          2.5.1                    pypi_0    pypi
flatbuffers               1.12                     pypi_0    pypi
freetype                  2.10.4               ha233b18_0  
fsspec                    0.9.0              pyhd3eb1b0_0  
future                    0.18.2                   py38_1  
futures                   3.0.5                    pypi_0    pypi
gast                      0.4.0                    pypi_0    pypi
gdal                      3.5.1                    pypi_0    pypi
get_terminal_size         1.0.0                h7520d66_0  
gettext                   0.21.0               h7535e17_0  
gevent                    21.1.2           py38h9ed2024_1  
gitdb                     4.0.9                    pypi_0    pypi
gitpython                 3.1.27                   pypi_0    pypi
glib                      2.68.1               hdf23fa2_0  
glob2                     0.7                pyhd3eb1b0_0  
gmp                       6.2.1                h23ab428_2  
gmpy2                     2.0.8            py38h6ef4df4_3  
google-api-core           2.7.1                    pypi_0    pypi
google-api-python-client  2.55.0                   pypi_0    pypi
google-auth               2.6.2                    pypi_0    pypi
google-auth-httplib2      0.1.0                    pypi_0    pypi
google-auth-oauthlib      0.5.2                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
googleapis-common-protos  1.56.0                   pypi_0    pypi
greenlet                  1.0.0            py38h23ab428_2  
grpcio                    1.46.1                   pypi_0    pypi
gtts                      2.2.4                    pypi_0    pypi
h5py                      2.10.0           py38h3134771_0  
hdf5                      1.10.4               hfa1e0ec_0  
heapdict                  1.0.1                      py_0  
html5lib                  1.1                        py_0  
httplib2                  0.20.4                   pypi_0    pypi
humanfriendly             10.0                     pypi_0    pypi
icu                       58.2                 h0a44026_3  
idna                      2.10               pyhd3eb1b0_0  
imageio                   2.9.0              pyhd3eb1b0_0  
imagesize                 1.2.0              pyhd3eb1b0_0  
importlib-metadata        4.11.3                   pypi_0    pypi
importlib_metadata        3.10.0               hd3eb1b0_0  
imutils                   0.5.4                    pypi_0    pypi
infinity                  1.5                      pypi_0    pypi
iniconfig                 1.1.1              pyhd3eb1b0_0  
intel-openmp              2021.2.0           hecd8cb5_564  
intervaltree              3.1.0                      py_0  
invoke                    1.6.0                    pypi_0    pypi
ipykernel                 5.3.4            py38h5ca1d4c_0  
ipython                   7.22.0           py38h01d92e1_0  
ipython_genutils          0.2.0              pyhd3eb1b0_1  
ipywidgets                7.6.3              pyhd3eb1b0_1  
isodate                   0.6.1                    pypi_0    pypi
isort                     5.8.0              pyhd3eb1b0_0  
itsdangerous              1.1.0              pyhd3eb1b0_0  
jaraco-context            4.1.2                    pypi_0    pypi
javaproperties            0.5.2                    pypi_0    pypi
jbig                      2.1                  h4d881f8_0  
jdcal                     1.4.1                      py_0  
jedi                      0.17.2           py38hecd8cb5_1  
jinja2                    2.11.3             pyhd3eb1b0_0  
jmespath                  0.10.0                   pypi_0    pypi
joblib                    1.0.1              pyhd3eb1b0_0  
jpeg                      9b                   he5867d9_2  
js2py                     0.71                     pypi_0    pypi
json5                     0.9.5                      py_0  
jsondiff                  1.3.1                    pypi_0    pypi
jsonschema                3.2.0                      py_2  
jupyter-packaging         0.7.12             pyhd3eb1b0_0  
jupyter_client            6.1.12             pyhd3eb1b0_0  
jupyter_console           6.4.0              pyhd3eb1b0_0  
jupyter_core              4.7.1            py38hecd8cb5_0  
jupyter_server            1.4.1            py38hecd8cb5_0  
jupyterlab_pygments       0.1.2                      py_0  
jupyterlab_server         2.4.0              pyhd3eb1b0_0  
jupyterlab_widgets        1.0.0              pyhd3eb1b0_1  
keras                     2.9.0                    pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
keyring                   22.3.0           py38hecd8cb5_0  
kiwisolver                1.3.1            py38h23ab428_0  
knack                     0.9.0                    pypi_0    pypi
krb5                      1.18.2               h75d18d8_0  
lazy-object-proxy         1.6.0            py38h9ed2024_0  
lcms2                     2.12                 hf1fd2bf_0  
libarchive                3.4.2                haa3ed63_0  
libclang                  14.0.1                   pypi_0    pypi
libcurl                   7.71.1               h8a08a2b_1  
libcxx                    10.0.0                        1  
libedit                   3.1.20210216         h9ed2024_1  
libev                     4.33                 haf1e3a3_0  
libffi                    3.3                  hb1e8313_2  
libgfortran               3.0.1                h93005f0_2  
libiconv                  1.16                 h1de35cc_0  
liblief                   0.10.1               h0a44026_0  
libllvm10                 10.0.1               h76017ad_5  
libpng                    1.6.37               ha441bb4_0  
libsodium                 1.0.18               h1de35cc_0  
libspatialindex           1.9.3                h23ab428_0  
libssh2                   1.9.0                ha12b0ac_1  
libtiff                   4.2.0                h87d7836_0  
libuv                     1.40.0               haf1e3a3_0  
libwebp-base              1.2.0                h9ed2024_0  
libxml2                   2.9.10               h7cdb67c_3  
libxslt                   1.1.34               h83b36ba_0  
llvm-openmp               10.0.0               h28b9765_0  
llvmlite                  0.36.0           py38he4411ff_4  
locket                    0.2.1            py38hecd8cb5_1  
lxml                      4.6.3            py38h26b266a_0  
lz4-c                     1.9.3                h23ab428_0  
lzo                       2.10                 haf1e3a3_2  
markdown                  3.3.7                    pypi_0    pypi
markupsafe                1.1.1            py38h1de35cc_1  
matplotlib                3.3.4            py38hecd8cb5_0  
matplotlib-base           3.3.4            py38h8b3ea08_0  
mccabe                    0.6.1                    py38_1  
mistune                   0.8.4           py38h1de35cc_1001  
mkl                       2021.2.0           hecd8cb5_269  
mkl-service               2.3.0            py38h9ed2024_1  
mkl_fft                   1.3.0            py38h4a7008c_2  
mkl_random                1.2.1            py38hb2f4e1b_2  
mock                      4.0.3              pyhd3eb1b0_0  
more-itertools            8.7.0              pyhd3eb1b0_0  
mpc                       1.1.0                h6ef4df4_1  
mpfr                      4.0.2                h9066e36_1  
mpmath                    1.2.1            py38hecd8cb5_0  
msal                      1.16.0                   pypi_0    pypi
msal-extensions           0.3.1                    pypi_0    pypi
msgpack-python            1.0.2            py38hf7b0b51_1  
msrest                    0.6.21                   pypi_0    pypi
msrestazure               0.6.4                    pypi_0    pypi
multipledispatch          0.6.0                    py38_0  
multitasking              0.0.10                   pypi_0    pypi
mypy_extensions           0.4.3                    py38_0  
navigator-updater         0.2.1                    py38_0  
nbclassic                 0.2.6              pyhd3eb1b0_0  
nbclient                  0.5.3              pyhd3eb1b0_0  
nbconvert                 6.0.7                    py38_0  
nbformat                  5.1.3              pyhd3eb1b0_0  
ncurses                   6.2                  h0a44026_1  
neat-python               0.92                     pypi_0    pypi
nest-asyncio              1.5.1              pyhd3eb1b0_0  
networkx                  2.5                        py_0  
nfl-data-py               0.3.0                    pypi_0    pypi
nltk                      3.6.1              pyhd3eb1b0_0  
nose                      1.3.7           pyhd3eb1b0_1006  
notebook                  6.3.0            py38hecd8cb5_0  
numba                     0.53.1           py38hb2f4e1b_0  
numexpr                   2.7.3            py38h5873af2_1  
numpy                     1.20.1           py38hd6e1bb9_0  
numpy-base                1.20.1           py38h585ceec_0  
numpydoc                  1.1.0              pyhd3eb1b0_1  
oauth2client              4.1.3                    pypi_0    pypi
oauthlib                  3.1.1                    pypi_0    pypi
olefile                   0.46                       py_0  
opencv-contrib-python     4.5.5.64                 pypi_0    pypi
opencv-python             4.5.5.64                 pypi_0    pypi
openpyxl                  3.0.7              pyhd3eb1b0_0  
openssl                   1.1.1k               h9ed2024_0  
opt-einsum                3.3.0                    pypi_0    pypi
packaging                 21.3                     pypi_0    pypi
pandas                    1.2.4            py38h23ab428_0  
pandoc                    2.12                 hecd8cb5_0  
pandocfilters             1.4.3            py38hecd8cb5_1  
paramiko                  2.9.2                    pypi_0    pypi
parso                     0.7.0                      py_0  
partd                     1.2.0              pyhd3eb1b0_0  
path                      15.1.2           py38hecd8cb5_0  
path.py                   12.5.0                        0  
pathlib2                  2.3.5            py38hecd8cb5_2  
pathspec                  0.10.0                   pypi_0    pypi
patsy                     0.5.2                    pypi_0    pypi
pcre                      8.44                 hb1e8313_0  
pep8                      1.7.1                    py38_0  
pexpect                   4.8.0              pyhd3eb1b0_3  
pickle-mixin              1.0.2                    pypi_0    pypi
pickleshare               0.7.5           pyhd3eb1b0_1003  
pillow                    8.2.0            py38h5270095_0  
pip                       21.3.1                   pypi_0    pypi
pipdeptree                2.2.1                    pypi_0    pypi
pipenv                    2022.8.24                pypi_0    pypi
pkginfo                   1.7.0            py38hecd8cb5_0  
platformdirs              2.4.1                    pypi_0    pypi
playsound                 1.3.0                    pypi_0    pypi
plotly                    5.8.1                    pypi_0    pypi
pluggy                    0.13.1           py38hecd8cb5_0  
ply                       3.11                     py38_0  
portalocker               1.7.1                    pypi_0    pypi
portaudio                 19.6.0               h647c56a_4  
prometheus_client         0.10.1             pyhd3eb1b0_0  
prompt-toolkit            3.0.17             pyh06a4308_0  
prompt_toolkit            3.0.17               hd3eb1b0_0  
protobuf                  3.19.4                   pypi_0    pypi
psutil                    5.8.0            py38h9ed2024_1  
ptyprocess                0.7.0              pyhd3eb1b0_2  
py                        1.10.0             pyhd3eb1b0_0  
py-lief                   0.10.1           py38haf313ee_0  
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pyaudio                   0.2.11           py38h1de35cc_2  
pycodestyle               2.6.0              pyhd3eb1b0_0  
pycosat                   0.6.3            py38h1de35cc_1  
pycparser                 2.20                       py_2  
pycurl                    7.43.0.6         py38hddc9c9b_0  
pydocstyle                6.0.0              pyhd3eb1b0_0  
pyerfa                    1.7.3            py38h9ed2024_0  
pyflakes                  2.2.0              pyhd3eb1b0_0  
pygithub                  1.55                     pypi_0    pypi
pygments                  2.8.1              pyhd3eb1b0_0  
pyjsparser                2.7.1                    pypi_0    pypi
pyjwt                     2.3.0                    pypi_0    pypi
pylint                    2.7.4            py38hecd8cb5_1  
pyls-black                0.4.6                hd3eb1b0_0  
pyls-spyder               0.3.2              pyhd3eb1b0_0  
pympler                   1.0.1                    pypi_0    pypi
pynacl                    1.4.0                    pypi_0    pypi
pyobjc                    8.5                      pypi_0    pypi
pyobjc-core               8.5                      pypi_0    pypi
pyobjc-framework-accessibility 8.5                      pypi_0    pypi
pyobjc-framework-accounts 8.5                      pypi_0    pypi
pyobjc-framework-addressbook 8.5                      pypi_0    pypi
pyobjc-framework-adservices 8.5                      pypi_0    pypi
pyobjc-framework-adsupport 8.5                      pypi_0    pypi
pyobjc-framework-applescriptkit 8.5                      pypi_0    pypi
pyobjc-framework-applescriptobjc 8.5                      pypi_0    pypi
pyobjc-framework-applicationservices 8.5                      pypi_0    pypi
pyobjc-framework-apptrackingtransparency 8.5                      pypi_0    pypi
pyobjc-framework-audiovideobridging 8.5                      pypi_0    pypi
pyobjc-framework-authenticationservices 8.5                      pypi_0    pypi
pyobjc-framework-automaticassessmentconfiguration 8.5                      pypi_0    pypi
pyobjc-framework-automator 8.5                      pypi_0    pypi
pyobjc-framework-avfoundation 8.5                      pypi_0    pypi
pyobjc-framework-avkit    8.5                      pypi_0    pypi
pyobjc-framework-businesschat 8.5                      pypi_0    pypi
pyobjc-framework-calendarstore 8.5                      pypi_0    pypi
pyobjc-framework-callkit  8.5                      pypi_0    pypi
pyobjc-framework-cfnetwork 8.5                      pypi_0    pypi
pyobjc-framework-classkit 8.5                      pypi_0    pypi
pyobjc-framework-cloudkit 8.5                      pypi_0    pypi
pyobjc-framework-cocoa    8.5                      pypi_0    pypi
pyobjc-framework-collaboration 8.5                      pypi_0    pypi
pyobjc-framework-colorsync 8.5                      pypi_0    pypi
pyobjc-framework-contacts 8.5                      pypi_0    pypi
pyobjc-framework-contactsui 8.5                      pypi_0    pypi
pyobjc-framework-coreaudio 8.5                      pypi_0    pypi
pyobjc-framework-coreaudiokit 8.5                      pypi_0    pypi
pyobjc-framework-corebluetooth 8.5                      pypi_0    pypi
pyobjc-framework-coredata 8.5                      pypi_0    pypi
pyobjc-framework-corehaptics 8.5                      pypi_0    pypi
pyobjc-framework-corelocation 8.5                      pypi_0    pypi
pyobjc-framework-coremedia 8.5                      pypi_0    pypi
pyobjc-framework-coremediaio 8.5                      pypi_0    pypi
pyobjc-framework-coremidi 8.5                      pypi_0    pypi
pyobjc-framework-coreml   8.5                      pypi_0    pypi
pyobjc-framework-coremotion 8.5                      pypi_0    pypi
pyobjc-framework-coreservices 8.5                      pypi_0    pypi
pyobjc-framework-corespotlight 8.5                      pypi_0    pypi
pyobjc-framework-coretext 8.5                      pypi_0    pypi
pyobjc-framework-corewlan 8.5                      pypi_0    pypi
pyobjc-framework-cryptotokenkit 8.5                      pypi_0    pypi
pyobjc-framework-devicecheck 8.5                      pypi_0    pypi
pyobjc-framework-dictionaryservices 8.5                      pypi_0    pypi
pyobjc-framework-discrecording 8.5                      pypi_0    pypi
pyobjc-framework-discrecordingui 8.5                      pypi_0    pypi
pyobjc-framework-diskarbitration 8.5                      pypi_0    pypi
pyobjc-framework-dvdplayback 8.5                      pypi_0    pypi
pyobjc-framework-eventkit 8.5                      pypi_0    pypi
pyobjc-framework-exceptionhandling 8.5                      pypi_0    pypi
pyobjc-framework-executionpolicy 8.5                      pypi_0    pypi
pyobjc-framework-externalaccessory 8.5                      pypi_0    pypi
pyobjc-framework-fileprovider 8.5                      pypi_0    pypi
pyobjc-framework-fileproviderui 8.5                      pypi_0    pypi
pyobjc-framework-findersync 8.5                      pypi_0    pypi
pyobjc-framework-fsevents 8.5                      pypi_0    pypi
pyobjc-framework-gamecenter 8.5                      pypi_0    pypi
pyobjc-framework-gamecontroller 8.5                      pypi_0    pypi
pyobjc-framework-gamekit  8.5                      pypi_0    pypi
pyobjc-framework-gameplaykit 8.5                      pypi_0    pypi
pyobjc-framework-imagecapturecore 8.5                      pypi_0    pypi
pyobjc-framework-imserviceplugin 8.5                      pypi_0    pypi
pyobjc-framework-inputmethodkit 8.5                      pypi_0    pypi
pyobjc-framework-installerplugins 8.5                      pypi_0    pypi
pyobjc-framework-instantmessage 8.5                      pypi_0    pypi
pyobjc-framework-intents  8.5                      pypi_0    pypi
pyobjc-framework-iosurface 8.5                      pypi_0    pypi
pyobjc-framework-ituneslibrary 8.5                      pypi_0    pypi
pyobjc-framework-kernelmanagement 8.5                      pypi_0    pypi
pyobjc-framework-latentsemanticmapping 8.5                      pypi_0    pypi
pyobjc-framework-launchservices 8.5                      pypi_0    pypi
pyobjc-framework-libdispatch 8.5                      pypi_0    pypi
pyobjc-framework-linkpresentation 8.5                      pypi_0    pypi
pyobjc-framework-localauthentication 8.5                      pypi_0    pypi
pyobjc-framework-mapkit   8.5                      pypi_0    pypi
pyobjc-framework-mediaaccessibility 8.5                      pypi_0    pypi
pyobjc-framework-medialibrary 8.5                      pypi_0    pypi
pyobjc-framework-mediaplayer 8.5                      pypi_0    pypi
pyobjc-framework-mediatoolbox 8.5                      pypi_0    pypi
pyobjc-framework-metal    8.5                      pypi_0    pypi
pyobjc-framework-metalkit 8.5                      pypi_0    pypi
pyobjc-framework-metalperformanceshaders 8.5                      pypi_0    pypi
pyobjc-framework-metalperformanceshadersgraph 8.5                      pypi_0    pypi
pyobjc-framework-mlcompute 8.5                      pypi_0    pypi
pyobjc-framework-modelio  8.5                      pypi_0    pypi
pyobjc-framework-multipeerconnectivity 8.5                      pypi_0    pypi
pyobjc-framework-naturallanguage 8.5                      pypi_0    pypi
pyobjc-framework-netfs    8.5                      pypi_0    pypi
pyobjc-framework-network  8.5                      pypi_0    pypi
pyobjc-framework-networkextension 8.5                      pypi_0    pypi
pyobjc-framework-notificationcenter 8.5                      pypi_0    pypi
pyobjc-framework-opendirectory 8.5                      pypi_0    pypi
pyobjc-framework-osakit   8.5                      pypi_0    pypi
pyobjc-framework-oslog    8.5                      pypi_0    pypi
pyobjc-framework-passkit  8.5                      pypi_0    pypi
pyobjc-framework-pencilkit 8.5                      pypi_0    pypi
pyobjc-framework-photos   8.5                      pypi_0    pypi
pyobjc-framework-photosui 8.5                      pypi_0    pypi
pyobjc-framework-preferencepanes 8.5                      pypi_0    pypi
pyobjc-framework-pushkit  8.5                      pypi_0    pypi
pyobjc-framework-quartz   8.5                      pypi_0    pypi
pyobjc-framework-quicklookthumbnailing 8.5                      pypi_0    pypi
pyobjc-framework-replaykit 8.5                      pypi_0    pypi
pyobjc-framework-safariservices 8.5                      pypi_0    pypi
pyobjc-framework-scenekit 8.5                      pypi_0    pypi
pyobjc-framework-screensaver 8.5                      pypi_0    pypi
pyobjc-framework-screentime 8.5                      pypi_0    pypi
pyobjc-framework-scriptingbridge 8.5                      pypi_0    pypi
pyobjc-framework-searchkit 8.5                      pypi_0    pypi
pyobjc-framework-security 8.5                      pypi_0    pypi
pyobjc-framework-securityfoundation 8.5                      pypi_0    pypi
pyobjc-framework-securityinterface 8.5                      pypi_0    pypi
pyobjc-framework-servicemanagement 8.5                      pypi_0    pypi
pyobjc-framework-social   8.5                      pypi_0    pypi
pyobjc-framework-soundanalysis 8.5                      pypi_0    pypi
pyobjc-framework-speech   8.5                      pypi_0    pypi
pyobjc-framework-spritekit 8.5                      pypi_0    pypi
pyobjc-framework-storekit 8.5                      pypi_0    pypi
pyobjc-framework-syncservices 8.5                      pypi_0    pypi
pyobjc-framework-systemconfiguration 8.5                      pypi_0    pypi
pyobjc-framework-systemextensions 8.5                      pypi_0    pypi
pyobjc-framework-uniformtypeidentifiers 8.5                      pypi_0    pypi
pyobjc-framework-usernotifications 8.5                      pypi_0    pypi
pyobjc-framework-usernotificationsui 8.5                      pypi_0    pypi
pyobjc-framework-videosubscriberaccount 8.5                      pypi_0    pypi
pyobjc-framework-videotoolbox 8.5                      pypi_0    pypi
pyobjc-framework-virtualization 8.5                      pypi_0    pypi
pyobjc-framework-vision   8.5                      pypi_0    pypi
pyobjc-framework-webkit   8.5                      pypi_0    pypi
pyodbc                    4.0.30           py38h0a44026_0  
pyopenssl                 20.0.1             pyhd3eb1b0_1  
pyparsing                 2.4.7              pyhd3eb1b0_0  
pyprind                   2.11.3                   pypi_0    pypi
pyqt                      5.9.2            py38h655552a_2  
pyreadr                   0.4.2                    pypi_0    pypi
pyrsistent                0.17.3           py38haf1e3a3_0  
pysmartdl                 1.3.4                    pypi_0    pypi
pysocks                   1.7.1                    py38_1  
pytesseract               0.3.9                    pypi_0    pypi
pytest                    6.2.3            py38hecd8cb5_2  
python                    3.8.8                h88f2d9e_5  
python-dateutil           2.8.1              pyhd3eb1b0_0  
python-jsonrpc-server     0.4.0                      py_0  
python-language-server    0.36.2             pyhd3eb1b0_0  
python-libarchive-c       2.9                pyhd3eb1b0_1  
python-snappy             0.6.1                    pypi_0    pypi
python.app                3                py38h9ed2024_0  
pytz                      2021.1             pyhd3eb1b0_0  
pytz-deprecation-shim     0.1.0.post0              pypi_0    pypi
pywavelets                1.1.1            py38haf1e3a3_2  
pyyaml                    5.4.1            py38h9ed2024_1  
pyzmq                     20.0.0           py38h23ab428_1  
qdarkstyle                2.8.1                      py_0  
qt                        5.9.7                h468cd18_1  
qtawesome                 1.0.2              pyhd3eb1b0_0  
qtconsole                 5.0.3              pyhd3eb1b0_0  
qtpy                      1.9.0                      py_0  
readline                  8.1                  h9ed2024_0  
regex                     2021.4.4         py38h9ed2024_0  
requests                  2.25.1             pyhd3eb1b0_0  
requests-oauthlib         1.3.0                    pypi_0    pypi
rich                      12.4.4                   pypi_0    pypi
ripgrep                   12.1.1                        0  
rope                      0.18.0                     py_0  
rsa                       4.8                      pypi_0    pypi
rtree                     0.9.7            py38hecd8cb5_1  
ruamel_yaml               0.15.100         py38h9ed2024_0  
s3transfer                0.5.0                    pypi_0    pypi
scikit-image              0.18.1           py38hb2f4e1b_0  
scikit-learn              0.24.1           py38hb2f4e1b_0  
scipy                     1.6.2            py38hd5f7400_1  
scp                       0.13.6                   pypi_0    pypi
screen                    1.0.1                    pypi_0    pypi
seaborn                   0.11.1             pyhd3eb1b0_0  
selenium                  3.141.0         py38h1de35cc_1001  
semver                    2.13.0                   pypi_0    pypi
send2trash                1.5.0              pyhd3eb1b0_1  
setuptools                52.0.0           py38hecd8cb5_0  
simplegeneric             0.8.1                    py38_2  
singledispatch            3.6.1           pyhd3eb1b0_1001  
sip                       4.19.8           py38h0a44026_0  
six                       1.15.0           py38hecd8cb5_0  
smmap                     5.0.0                    pypi_0    pypi
sniffio                   1.2.0            py38hecd8cb5_1  
snowballstemmer           2.1.0              pyhd3eb1b0_0  
sortedcollections         2.1.0              pyhd3eb1b0_0  
sortedcontainers          2.3.0              pyhd3eb1b0_0  
sounddevice               0.4.4                    pypi_0    pypi
soundfile                 0.10.3.post1             pypi_0    pypi
soupsieve                 2.2.1              pyhd3eb1b0_0  
sphinx                    4.0.1              pyhd3eb1b0_0  
sphinxcontrib             1.0                      py38_1  
sphinxcontrib-applehelp   1.0.2              pyhd3eb1b0_0  
sphinxcontrib-devhelp     1.0.2              pyhd3eb1b0_0  
sphinxcontrib-htmlhelp    1.0.3              pyhd3eb1b0_0  
sphinxcontrib-jsmath      1.0.1              pyhd3eb1b0_0  
sphinxcontrib-qthelp      1.0.3              pyhd3eb1b0_0  
sphinxcontrib-serializinghtml 1.1.4              pyhd3eb1b0_0  
spyder-kernels            1.10.2           py38hecd8cb5_0  
sqlalchemy                1.4.7            py38h9ed2024_0  
sqlite                    3.35.4               hce871da_0  
sqlparse                  0.4.2                    pypi_0    pypi
sshtunnel                 0.1.5                    pypi_0    pypi
statsmodels               0.13.1                   pypi_0    pypi
tabulate                  0.8.9                    pypi_0    pypi
tbb                       2020.3               h879752b_0  
tblib                     1.7.0                      py_0  
tenacity                  8.0.1                    pypi_0    pypi
tensorboard               2.9.0                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.1                    pypi_0    pypi
tensorflow-estimator      2.9.0                    pypi_0    pypi
tensorflow-io-gcs-filesystem 0.26.0                   pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
terminado                 0.9.4            py38hecd8cb5_0  
testpath                  0.4.4              pyhd3eb1b0_0  
textdistance              4.2.1              pyhd3eb1b0_0  
threadpoolctl             2.1.0              pyh5ca1d4c_0  
three-merge               0.1.1              pyhd3eb1b0_0  
tifffile                  2020.10.1        py38h0cf3a3e_2  
tk                        8.6.10               hb0a8c7a_0  
toml                      0.10.2             pyhd3eb1b0_0  
tomli                     2.0.1                    pypi_0    pypi
toolz                     0.11.1             pyhd3eb1b0_0  
tornado                   6.1              py38h9ed2024_0  
tqdm                      4.59.0             pyhd3eb1b0_1  
traces                    0.6.0                    pypi_0    pypi
traitlets                 5.0.5              pyhd3eb1b0_0  
typed-ast                 1.4.2            py38h9ed2024_1  
typing-extensions         4.2.0                    pypi_0    pypi
tzdata                    2022.1                   pypi_0    pypi
tzlocal                   4.2                      pypi_0    pypi
ujson                     4.0.2            py38h23ab428_0  
unicodecsv                0.14.1                   py38_0  
unixodbc                  2.3.9                haf1e3a3_0  
uritemplate               4.1.1                    pypi_0    pypi
urllib3                   1.26.4             pyhd3eb1b0_0  
validators                0.20.0                   pypi_0    pypi
virtualenv                20.13.0                  pypi_0    pypi
virtualenv-clone          0.5.7                    pypi_0    pypi
watchdog                  1.0.2            py38h9ed2024_1  
wcwidth                   0.2.5                      py_0  
webencodings              0.5.1                    py38_1  
websocket-client          0.56.0                   pypi_0    pypi
websockets                10.1                     pypi_0    pypi
werkzeug                  1.0.1              pyhd3eb1b0_0  
wheel                     0.36.2             pyhd3eb1b0_0  
widgetsnbextension        3.5.1                    py38_0  
wrapt                     1.12.1           py38haf1e3a3_1  
wurlitzer                 2.1.0            py38hecd8cb5_0  
xlwt                      1.3.0                    py38_0  
xmltodict                 0.12.0                     py_0  
xz                        5.2.5                h1de35cc_0  
yaml                      0.2.5                haf1e3a3_0  
zeromq                    4.3.4                h23ab428_0  
zict                      2.0.0              pyhd3eb1b0_0  
zipp                      3.4.1              pyhd3eb1b0_0  
zlib                      1.2.11               h1de35cc_3  
zope                      1.0                      py38_1  
zope.event                4.5.0                    py38_0  
zope.interface            5.3.0            py38h9ed2024_0  
zstd                      1.4.5                h41d2c2f_0

**Comment by nickdif5191** [comment]
I was having a similar issue, getting the same error when trying to run import_pbp_data. However, I was getting a different error when trying to run see_pbp_cols, the error read 

"ImportError: Unable to find a usable engine; tried using: 'pyarrow', 'fastparquet' "

I simply ran the following code in my conda prompt (it didn't fix the problem when I ran it in my Windows terminal):
"
pip install pyarrow

pip install fastparquet
"

Now see_pbp_cols and import_pbp_data both work for me.

Maybe give this a shot. Good luck!!

**Comment by cooperdff** [comment]
Hi @rjthompson2 - did the suggestion above work for you?

**Comment by JaWeilBaum** [comment]
Hi @rjthompson2 & @cooperdff, 
I had the same issue. As it turns out @rjthompson2 might be using macOS and this seems to be a common issue, that there is an certificate issue. I fixed it for myself by running the following in the command line:

```bash
/Applications/Python\ 3.9/Install\ Certificates.command
```

I hope this solves the issue for you @rjthompson2!

@cooperdff Thanks for the downloader, will use it now! How ever I found out that pandas is not propagating the issue of the SSL error. If you execute the command in `__init__.py` on line 114 you will get the error, but it is not shown to the user. The SSL error is only shown when you use `nfl.see_pbp_cols()`
